src/main/java/appeng/core/features/AEFeature.java

```diff
@@ -81,15 +87,28 @@ public boolean isVisible()
 	ExportBus( Constants.CATEGORY_NETWORK_BUSES ),
 	StorageBus( Constants.CATEGORY_NETWORK_BUSES ),
 	PartConversionMonitor( Constants.CATEGORY_NETWORK_BUSES ),
+	ToggleBus( Constants.CATEGORY_NETWORK_BUSES ),
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00115)